apps/core/scenes/search/Placepickers.js

```diff
@@ -9,36 +9,39 @@ import {
 import { Icon, StyleSheet } from '@kiwicom/universal-components';
 import { defaultTokens } from '@kiwicom/orbit-design-tokens';
 
+import { withSearchContext, type SearchContextState } from './SearchContext';
+
 type Props = {|
   +travelFrom: string,
   +travelTo: string,
-  +handlePlacePress: () => void,
   +handlePlaceSwitchPress: () => void,
 |};
 
-export default function Placepickers(props: Props) {
-  return (
-    <View>
-      <TripInput
-        onPress={props.handlePlacePress}
-        label="From"
-        icon={<Icon name="airplane-takeoff" />}
-        value={props.travelFrom}
-      />
-      <TouchableWithoutFeedback onPress={props.handlePlaceSwitchPress}>
-        <View style={styles.placeSwitch}>
-          <Icon name="replace" color="#7F91A8" />
-        </View>
-      </TouchableWithoutFeedback>
-      <TripInput
-        onPress={props.handlePlacePress}
-        label="To"
-        icon={<Icon name="airplane-landing" />}
-        value={props.travelTo}
-      />
-    </View>
-  );
-}
+const handlePlacePress = () => {
```

Comment:
Should this be as a property of class Placpickers instead? Or will it not be re-created on ervery re-render?

Reply:
This is defined in the file, so this function is created only once. 

When we get a proper placepicker, I assume we will wrap this component in a `withNavigation` and handle the on click even in a `class Placepickers`